src/SimulationRunner.cc

```diff
@@ -511,7 +540,7 @@ void SimulationRunner::ProcessSystemQueue()
   // If additional systems were added, recreate the worker threads.
   if (pending > 0)
   {
-    igndbg << "Creating PostUpdate worker threads: "
+    ignwarn << "Creating PostUpdate worker threads: "
```

Comment:
Careful with this log level changes. Please revert those when promoting this from Draft to PR